_assets/javascripts/components/resi-player.js

```diff
@@ -54,6 +54,11 @@ const refreshPageForServiceStart = (hours, minutes, seconds) => {
   }, timeout);
 };
 
+// For OnDemand Card
+if (isServiceTime() && document.getElementById('ondemand-cta')) {
+  document.getElementById('ondemand-cta').remove();
+} 
+
```

Comment:
So I have realized a problem with this solution... There is a 10 minute space between services where this block may show up. This will be an issue if they have not posted the new video until 11.

My suggestion is to create a similar function to the `isServiceTime()` that is specifically for this on demand block. We are going to want it to want this block to not show up between 8:25 and 1:30 on Sundays